linajea/candidate_database.py

```diff
@@ -62,6 +62,14 @@ def __init__(
                 position_attribute=['t', 'z', 'y', 'x'],
                 endpoint_names=endpoint_names
                 )
+        if mode == 'w':
+            try:
+                self._MongoDbGraphProvider__connect()
+                self._MongoDbGraphProvider__open_db()
+                params_collection = self.database['parameters']
+                params_collection.drop()
+            finally:
+                self._MongoDbGraphProvider__disconnect()
```

Comment:
mode doesn't seem to be used anywhere else, has it been used somewhere previously? If not, we could think about if this is the best way to do it. And there is a mismatch between the code and the doc string.

Reply:
Mode is used in the super initializer (MongoDbGraphProvider)

Reply:
I fixed the docstring to include the parameters db being dropped in w mode. Since the rest of mode functionality is part of the daisy graph provider, I think we are stuck with it for now. (I mean, I am the person in charge of that code, and I agree it doesn't make much sense, but I haven't had a compelling reason to change it).